pkg/assembler/graphql/schema/hasSBOM.graphql

```diff
@@ -30,38 +34,72 @@ type HasSBOM {
   id: ID!
   subject: PackageOrSource!
   uri: String!
-  annotation: String!
+  algorithm: String!
```

Comment:
@mihaimaruseac @mihaimaruseac wanted feedback if this meets all our needs before proceeding with the implementation.